dwave_networkx/drawing/qubit_layout.py

```diff
@@ -240,6 +259,16 @@ def draw_embedding(G, layout, emb, embedded_graph=None, interaction_edges=None,
         the same vertices in G), and the drawing will display these overlaps as
         concentric circles.
 
+    line_plot : boolean (optional, default False)
+        If line_plot is True, then qubits are drawn as line segments, and edges
+        are drawn either as line segments between qubits, or as circles where
+        two qubits overlap.  In this drawing style, the interpretation the width
+        and node_size parameters (provided in kwargs) determines the area of the
+        circles, and line widths, respectively.  Qubit line segments are given
+        twice the width of edges.  Layout should be a dict of the form
+        {node: ((x0, y0), (y0, x0)), ...} -- instead of coordinates, the nodes
+        are associated with endpoints of n-dimensional line segments.
```

Comment:
```suggestion
    line_plot : boolean (optional, default False)
        If ``line_plot`` is ``True``, qubits are drawn as line segments and edges
        are drawn as either line segments between qubits or circles where
        two qubits overlap.  In this drawing style, the interpretation of the ``width``
        and ``node_size`` parameters (provided in ``kwargs``) determines the area of the
        circles, and line widths, respectively.  Qubit line segments are given
        twice the width of edges.  Layout should be a dict of the form
        ``{node: ((x0, y0), (y0, x0)), ...}``; instead of coordinates, the nodes
        are associated with endpoints of n-dimensional line segments.
```